docs/docker-image-details.md

```diff
@@ -44,6 +42,24 @@ Changelog
     - `8090` Gravitino Web UI
     - `9001` Iceberg REST service
 
+## Gravitino Iceberg REST server Docker image
+
+You can deploy the standalone Gravitino Iceberg REST server with the Docker image.
+
+Container startup commands
+
+```shell
+docker run --rm -d -p 9001:9001 datastrato/gravitino-iceberg-rest-server
+```
+
+Changelog
+
+- gravitino-iceberg-rest-server:0.6.0
```

Comment:
Does the version of `gravitino-iceberg-rest-server` goes along with Gravitino version?

Reply:
yes